### ISSUE TITLE
Get paid to View Adult Content XXX

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#4752]
Migrated from [rt.perl.org#8135](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=8135) (status was 'resolved')

Searchable as RT8135$

**Comment by p5pRT** [comment]
### From Bigwheel@wizeonIine.worldonline.co.uk

Hi \,
It's a Fact that over 70% of Adult surfers go LOOKING for Porn
Well now here's a twist How about getting Paid to View Porn \!\!

How? Whats the Catch ? 

Here's How\, Simply join me in offering FREE Access 24/7 to 40 Adult websites\,

Whats the Catch well really there is no Catch\, If you join our exclusive club for $25\.00 a month you will get FREE Access to HOT Adult Content 24/7\,

You Can make this offer to others and when they join they will also Pay $25\.00 a Month\,
You will be paid a Commission of 50% of ALL your Referred Members\, So 2 members gives you FREE Access any more and you will be paid 50% of all your referral members Fees\.

Ten new  Members a Month referred by you  will compound over 12 months to an income of Thousands of Dollars 
You do the Math 
I have at present 125 Members refferred and rising daily 
My Income last Month was in excess of 

In Summary here's the deal You get Unlimited Access to over 40 Adult Sites\, you get Paid 50% Commision for referring New Members and a Residual Ongoing income from it\!\!

Please note&#8203;: Only the serious enquiries will be noted you MUST ONLY put the words  'Please send me to your website'  in the subject line otherwise your mail "WILL BE REJECTED"

If you want to Join Me in this Brilliant Business email me for full details goto&#8203;:
 sfpteam@&#8203;wizonline\.worldonline\.co\.uksubject=?Please send me to your website
